Update `pnpm clean` script to include root directory and missing packages

cindyledev opened the pull request:
<!--
Thanks for sending a pull request!
- if this is your first time, please ensure you have read through our contributor guide: https://github.com/Seneca-CDOT/telescope/blob/master/docs/CONTRIBUTING.md
-->

## Issue This PR Addresses

<!--
1. Automatically close the issue when this PR is merged
    USAGE: Fixes #<issue number>
2. If your PR addresses an issue but does not close it
    USAGE: #<issue number> <reason>(i.e. This issue was worked on by @user and myself and his PR should close the issue.)
-->

## Type of Change

<!-- bug fix, feature, documentation, UI, etc. -->

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

<!-- Please add a detailed description of what this PR does and why it is needed -->

Currently, if we run `pnpm clean` in the root directory, it does not delete the `node_modules` directory in root as expected. 

The two commands for the `clean` script do very much the same thing, which is to delete directories and files

```
// package.json
"scripts": {
    ...
    "clean": "pnpm turbo run clean && pnpm -r exec rm -rf node_modules",
    ...
}
```

1. `pnpm -r exec rm -rf node_modules` deletes only `node_modules` directories in our pnpm workspaces. There are a total of 18 workspace projects (including root)

2. `pnpm turbo run clean` runs all the `clean` scripts in the following directory:
- `src/web/app` (telescope-frontend)
- `src/web/docusaurus` (telescope-docs)
- `src/api/status` (status-service)
- `src/api/image` (image-service)

That's right. There are only four packages with the `clean` script. The reason only these four packages had `clean` scripts was that we needed to delete `out`, `build`, etc directories on top of the `node_modules` directory.

This PR allows us to use Turborepo to run `pnpm clean` within our project scope exactly the same way we use Turborepo to run `pnpm lint` and `pnpm test`. It further decouples our packages so we can use `pnpm clean` in the individual packages and not rely on `pnpm clean` in the root directory. However, in most cases, we want to use it in the root directory, but now, we can do both.

## Steps to test the PR

<!-- Please add steps to build the changes in your PR locally so that peers can test your PR
    Example:
    - `npm install`
    - Go to '...'
    - Click on '....'
    - Scroll down to '....'
    - See error

    Please remember, the more clear you are, the faster your PR will be reviewed and merged.
 -->

To test this PR locally:
1. Add my repository as a remote: `git remote add cindyledev https://github.com/cindyledev/telescope.git`
2. Fetch my branches: `git fetch cindyledev`
3. Checkout my branch: `git checkout fix-clean-script`
4. Install dependencies using pnpm in root: `pnpm install`
5. When you run `pnpm clean` in the root directory, you should see the following output

```
 Tasks:    17 successful, 17 total
Cached:    0 cached, 17 total
  Time:    8.702s
```

The reason it's 17,and not 18 is because the root directory is not included in the count

## Checklist

<!-- Before submitting a PR, address each item -->

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

RC-Lee commented:
This is neat, I always thought it was my potato's fault that `pnpm clean` never worked for me. 

Unfortunately I still ran into an error
```
command (src\api\image) pnpm run clean exited (1)

 Tasks:    10 successful, 11 total
Cached:    0 cached, 11 total
  Time:    8.483s

 ELIFECYCLE  Command failed with exit code 1.
```

cindyledev commented:
> 

So apparently, `rm -rf` doesn't work on Windows machines...

*shakes fist at ceiling*

Thoughts on using the [rimraf](https://www.npmjs.com/package/rimraf)?

humphd commented:
rimraf is the preferred way to do this, yes.

cindyledev commented:
I'm not sure why the Vercel check failed here.

@RC-Lee Can your potato test and review this PR? 🥔

cindyledev commented:
> Vercel failure log
> ```
> 
> [10:13:32.684] Cloning github.com/Seneca-CDOT/telescope (Branch: fix-clean-script, Commit: 005f578)
> [10:13:34.524] Cloning completed: 1.840s
> [10:13:35.093] Looking up build cache...
> [10:13:42.109] Build cache downloaded [79.84 MB]: 6790.541ms
> [10:13:42.177] Running "vercel build"
> [10:13:42.925] Vercel CLI 28.4.10-6fa5d8b
> [10:13:43.345] Running "install" command: `pnpm install`...
> [10:13:44.296] Progress: resolved 1, reused 0, downloaded 0, added 0
> [10:13:45.297] Progress: resolved 13, reused 0, downloaded 8, added 0
> [10:13:46.298] Progress: resolved 21, reused 0, downloaded 16, added 0
> [10:13:47.310] Progress: resolved 33, reused 0, downloaded 25, added 0
> [10:13:48.313] Progress: resolved 37, reused 0, downloaded 31, added 0
> [10:13:49.315] Progress: resolved 42, reused 0, downloaded 38, added 0
> [10:13:50.341] Progress: resolved 43, reused 0, downloaded 42, added 0
> [10:13:51.347] Progress: resolved 46, reused 0, downloaded 43, added 0
> [10:13:52.352] Progress: resolved 46, reused 0, downloaded 45, added 0
> [10:13:53.358] Progress: resolved 47, reused 0, downloaded 45, added 0
> [10:13:54.358] Progress: resolved 47, reused 0, downloaded 46, added 0
> [10:13:55.364] Progress: resolved 48, reused 0, downloaded 46, added 0
> [10:13:57.380] Progress: resolved 48, reused 0, downloaded 47, added 0
> [10:13:58.583] Progress: resolved 49, reused 0, downloaded 47, added 0
> [10:13:59.584] Progress: resolved 50, reused 0, downloaded 47, added 0
> [10:14:00.585] Progress: resolved 50, reused 0, downloaded 48, added 0
> [10:14:01.589] Progress: resolved 67, reused 0, downloaded 50, added 0
> [10:14:02.601] Progress: resolved 122, reused 0, downloaded 99, added 0
> [10:14:03.612] Progress: resolved 159, reused 0, downloaded 137, added 0
> [10:14:04.617] Progress: resolved 221, reused 0, downloaded 200, added 0
> [10:14:05.624] Progress: resolved 266, reused 0, downloaded 244, added 0
> [10:14:06.627] Progress: resolved 324, reused 0, downloaded 301, added 0
> [10:14:07.632] Progress: resolved 397, reused 0, downloaded 373, added 0
> [10:14:08.638] Progress: resolved 502, reused 0, downloaded 478, added 0
> [10:14:09.640] Progress: resolved 637, reused 0, downloaded 613, added 0
> [10:14:10.647] Progress: resolved 692, reused 0, downloaded 678, added 0
> [10:14:11.126] Already up-to-date
> [10:14:11.647] Progress: resolved 692, reused 0, downloaded 679, added 0, done
> [10:14:12.299] 
> [10:14:12.300] devDependencies:
> [10:14:12.300] + rimraf 3.0.2
> [10:14:12.300] 
> [10:14:12.335] Detected Next.js version: 12.1.6
> [10:14:12.394] Running "pnpm run build"
> [10:14:13.062] 
> [10:14:13.062] > @senecacdot/telescope-frontend@3.0.0 build /vercel/path1
> [10:14:13.062] > next build && next export
> [10:14:13.062] 
> [10:14:14.726] Trying to load missing ENV variables from /.env
> [10:14:14.727] Trying to load missing ENV variables from /config/env.development
> [10:14:14.728] Using NEXT_PUBLIC_WEB_URL=https://telescope-h37b2so8z-humphd.vercel.app
> [10:14:14.728] Using NEXT_PUBLIC_IMAGE_URL=https://dev.api.telescope.cdot.systems/v1/image
> [10:14:14.728] Using NEXT_PUBLIC_POSTS_URL=https://dev.api.telescope.cdot.systems/v1/posts
> [10:14:14.728] Using NEXT_PUBLIC_SSO_URL=https://dev.api.telescope.cdot.systems/v1/auth
> [10:14:14.729] Using NEXT_PUBLIC_SEARCH_URL=https://dev.api.telescope.cdot.systems/v1/search
> [10:14:14.729] Using NEXT_PUBLIC_FEED_DISCOVERY_URL=https://dev.api.telescope.cdot.systems/v1/feed-discovery
> [10:14:14.729] Using NEXT_PUBLIC_STATUS_URL=https://dev.api.telescope.cdot.systems/v1/status
> [10:14:14.729] Using NEXT_PUBLIC_GIT_COMMIT=undefined
> [10:14:14.729] Using NEXT_PUBLIC_SUPABASE_URL=https://dev.api.telescope.cdot.systems/v1/supabase
> [10:14:14.730] Using NEXT_PUBLIC_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
> [10:14:14.730] Using NEXT_PUBLIC_DEPENDENCY_DISCOVERY_URL=https://dev.api.telescope.cdot.systems/v1/dependency-discovery
> [10:14:14.732] warn  - SWC minify release candidate enabled. https://nextjs.org/docs/messages/swc-minify-enabled
> [10:14:14.794] info  - Checking validity of types...
> [10:14:21.305] error - ESLint: Failed to load plugin 'react' declared in '.eslintrc.js » @senecacdot/eslint-config-telescope': Cannot find module 'eslint-plugin-react' Require stack: - /vercel/path1/__placeholder__.js Referenced from: /vercel/path1/node_modules/.pnpm/@senecacdot+eslint-config-telescope@1.1.0_eslint@7.32.0/node_modules/@senecacdot/eslint-config-telescope/index.js
> [10:14:21.309] info  - Creating an optimized production build...
> [10:14:21.373] > [PWA] Compile client (static)
> [10:14:21.373] > [PWA] Auto register service worker with: /vercel/path1/node_modules/.pnpm/next-pwa@5.4.6_b9348bf848bb7cd32c9b4304596174f6/node_modules/next-pwa/register.js
> [10:14:21.374] > [PWA] Service worker: /vercel/path1/public/sw.js
> [10:14:21.374] > [PWA]   url: /sw.js
> [10:14:21.375] > [PWA]   scope: /
> [10:14:21.674] > [PWA] Compile server
> [10:14:21.686] > [PWA] Compile server
> [10:14:55.000] info  - Compiled successfully
> [10:14:55.000] info  - Collecting page data...
> [10:15:00.667] info  - Generating static pages (0/5)
> [10:15:00.761] info  - Generating static pages (1/5)
> [10:15:00.774] info  - Generating static pages (2/5)
> [10:15:00.783] info  - Generating static pages (3/5)
> [10:15:00.864] info  - Generating static pages (5/5)
> [10:15:00.880] info  - Finalizing page optimization...
> [10:15:00.885] 
> [10:15:00.909] Page                                       Size     First Load JS
> [10:15:00.909] ┌ ○ /                                      11 kB           195 kB
> [10:15:00.909] ├   /_app                                  0 B             109 kB
> [10:15:00.909] ├ ○ /404                                   179 B           109 kB
> [10:15:00.910] ├ λ /dependencies                          30.8 kB         245 kB
> [10:15:00.910] ├ ○ /error                                 3.53 kB         133 kB
> [10:15:00.910] ├ λ /search                                4.66 kB         241 kB
> [10:15:00.910] └ ○ /signup                                42.5 kB         194 kB
> [10:15:00.910] + First Load JS shared by all              109 kB
> [10:15:00.910]   ├ chunks/framework-248ddb2458fac65d.js   42.2 kB
> [10:15:00.911]   ├ chunks/main-72922e6a2d43303e.js        31.1 kB
> [10:15:00.911]   ├ chunks/pages/_app-6075d7213a59e427.js  33.5 kB
> [10:15:00.911]   ├ chunks/webpack-2126900e39a57681.js     1.78 kB
> [10:15:00.911]   └ css/494c39f3d1bf8d97.css               1.55 kB
> [10:15:00.911] 
> [10:15:00.911] λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
> [10:15:00.911] ○  (Static)  automatically rendered as static HTML (uses no initial props)
> [10:15:00.912] 
> [10:15:02.303] Trying to load missing ENV variables from /.env
> [10:15:02.304] Trying to load missing ENV variables from /config/env.development
> [10:15:02.305] Using NEXT_PUBLIC_WEB_URL=https://telescope-h37b2so8z-humphd.vercel.app
> [10:15:02.305] Using NEXT_PUBLIC_IMAGE_URL=https://dev.api.telescope.cdot.systems/v1/image
> [10:15:02.305] Using NEXT_PUBLIC_POSTS_URL=https://dev.api.telescope.cdot.systems/v1/posts
> [10:15:02.305] Using NEXT_PUBLIC_SSO_URL=https://dev.api.telescope.cdot.systems/v1/auth
> [10:15:02.305] Using NEXT_PUBLIC_SEARCH_URL=https://dev.api.telescope.cdot.systems/v1/search
> [10:15:02.306] Using NEXT_PUBLIC_FEED_DISCOVERY_URL=https://dev.api.telescope.cdot.systems/v1/feed-discovery
> [10:15:02.306] Using NEXT_PUBLIC_STATUS_URL=https://dev.api.telescope.cdot.systems/v1/status
> [10:15:02.306] Using NEXT_PUBLIC_GIT_COMMIT=undefined
> [10:15:02.306] Using NEXT_PUBLIC_SUPABASE_URL=https://dev.api.telescope.cdot.systems/v1/supabase
> [10:15:02.306] Using NEXT_PUBLIC_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
> [10:15:02.307] Using NEXT_PUBLIC_DEPENDENCY_DISCOVERY_URL=https://dev.api.telescope.cdot.systems/v1/dependency-discovery
> [10:15:02.309] warn  - SWC minify release candidate enabled. https://nextjs.org/docs/messages/swc-minify-enabled
> [10:15:02.499] info  - using build directory: /vercel/path1/.next
> [10:15:02.504] info  - Copying "static build" directory
> [10:15:02.510] info  - No "exportPathMap" found in "/vercel/path1/next.config.js". Generating map from "./pages"
> [10:15:02.515] info  - Launching 3 workers
> [10:15:02.516] info  - Exporting (0/6)
> [10:15:02.516] info  - Copying "public" directory
> [10:15:03.301] info  - Exporting (1/6)
> [10:15:03.380] info  - Exporting (2/6)
> [10:15:03.386] info  - Exporting (4/6)
> [10:15:03.793] 
> [10:15:03.794] Error occurred prerendering page "/dependencies". Read more: https://nextjs.org/docs/messages/prerender-error
> [10:15:03.794] Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
> [10:15:03.794] Require stack:
> [10:15:03.794] - /vercel/path1/node_modules/.pnpm/@mui+private-classnames@5.7.0/node_modules/@mui/private-classnames/node/index.js
> [10:15:03.794] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/composeClasses/index.js
> [10:15:03.795] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/BadgeUnstyled/BadgeUnstyled.js
> [10:15:03.795] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/BadgeUnstyled/index.js
> [10:15:03.795] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/index.js
> [10:15:03.796] - /vercel/path1/node_modules/.pnpm/@mui+material@5.7.0_78e61854d81b39da5e9ad4ef04b13775/node_modules/@mui/material/node/IconButton/IconButton.js
> [10:15:03.796] - /vercel/path1/node_modules/.pnpm/@mui+material@5.7.0_78e61854d81b39da5e9ad4ef04b13775/node_modules/@mui/material/node/IconButton/index.js
> [10:15:03.796] - /vercel/path1/.next/server/pages/dependencies.js
> [10:15:03.796] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/server/require.js
> [10:15:03.797] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/server/load-components.js
> [10:15:03.797] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/export/worker.js
> [10:15:03.798] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/compiled/jest-worker/processChild.js
> [10:15:03.798]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
> [10:15:03.798]     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
> [10:15:03.799]     at Module.require (internal/modules/cjs/loader.js:974:19)
> [10:15:03.799]     at require (internal/modules/cjs/helpers.js:101:18)
> [10:15:03.799]     at Object. (/vercel/path1/node_modules/.pnpm/@mui+private-classnames@5.7.0/node_modules/@mui/private-classnames/node/index.js:8:30)
> [10:15:03.800]     at Module._compile (internal/modules/cjs/loader.js:1085:14)
> [10:15:03.800]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
> [10:15:03.800]     at Module.load (internal/modules/cjs/loader.js:950:32)
> [10:15:03.801]     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
> [10:15:03.801]     at Module.require (internal/modules/cjs/loader.js:974:19)
> [10:15:03.959] 
> [10:15:03.959] Error occurred prerendering page "/search". Read more: https://nextjs.org/docs/messages/prerender-error
> [10:15:03.959] Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
> [10:15:03.959] Require stack:
> [10:15:03.959] - /vercel/path1/node_modules/.pnpm/@mui+private-classnames@5.7.0/node_modules/@mui/private-classnames/node/index.js
> [10:15:03.959] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/composeClasses/index.js
> [10:15:03.960] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/BadgeUnstyled/BadgeUnstyled.js
> [10:15:03.960] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/BadgeUnstyled/index.js
> [10:15:03.960] - /vercel/path1/node_modules/.pnpm/@mui+base@5.0.0-alpha.80_569957a51d469c962fcce166a59b9f78/node_modules/@mui/base/node/index.js
> [10:15:03.961] - /vercel/path1/node_modules/.pnpm/@mui+material@5.7.0_78e61854d81b39da5e9ad4ef04b13775/node_modules/@mui/material/node/styles/createTheme.js
> [10:15:03.964] - /vercel/path1/node_modules/.pnpm/@mui+material@5.7.0_78e61854d81b39da5e9ad4ef04b13775/node_modules/@mui/material/node/styles/index.js
> [10:15:03.964] - /vercel/path1/node_modules/.pnpm/@mui+material@5.7.0_78e61854d81b39da5e9ad4ef04b13775/node_modules/@mui/material/node/index.js
> [10:15:03.964] - /vercel/path1/.next/server/pages/search.js
> [10:15:03.965] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/server/require.js
> [10:15:03.965] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/server/load-components.js
> [10:15:03.965] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/export/worker.js
> [10:15:03.965] - /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/compiled/jest-worker/processChild.js
> [10:15:03.965]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
> [10:15:03.965]     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
> [10:15:03.965]     at Module.require (internal/modules/cjs/loader.js:974:19)
> [10:15:03.966]     at require (internal/modules/cjs/helpers.js:101:18)
> [10:15:03.966]     at Object. (/vercel/path1/node_modules/.pnpm/@mui+private-classnames@5.7.0/node_modules/@mui/private-classnames/node/index.js:8:30)
> [10:15:03.966]     at Module._compile (internal/modules/cjs/loader.js:1085:14)
> [10:15:03.966]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
> [10:15:03.966]     at Module.load (internal/modules/cjs/loader.js:950:32)
> [10:15:03.967]     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
> [10:15:03.967]     at Module.require (internal/modules/cjs/loader.js:974:19)
> [10:15:03.967] info  - Exporting (6/6)
> [10:15:03.968] Error: Export encountered errors on following paths:
> [10:15:03.969] 	/dependencies
> [10:15:03.969] 	/search
> [10:15:03.969]     at /vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/export/index.js:398:19
> [10:15:03.969]     at processTicksAndRejections (internal/process/task_queues.js:95:5)
> [10:15:03.970]     at async Span.traceAsyncFn (/vercel/path1/node_modules/.pnpm/next@12.1.6_9b0027aae606c22bf55a7fd89836f009/node_modules/next/dist/trace/trace.js:79:20)
> [10:15:03.985]  ELIFECYCLE  Command failed with exit code 1.
> [10:15:04.006] Error: Command "pnpm run build" exited with 1
> ```

Hmm I guess #3650 is still a problem. I'm gonna reopen that ticket

RC-Lee commented:
> I'm not sure why the Vercel check failed here.
> 
> @RC-Lee Can your potato test and review this PR? 🥔

I get an error near the end when it tries to clean the root. 
```
@senecacdot/image-service:clean: The system cannot find the path specified.
@senecacdot/parser-service:clean: The system cannot find the path specified.
@senecacdot/eslint-config-telescope:clean: The system cannot find the path specified.
migrate:clean: The system cannot find the path specified.
@senecacdot/satellite:clean: The system cannot find the path specified.
@senecacdot/posts-service:clean: The system cannot find the path specified.
 Tasks:    17 successful, 17 total
Cached:    0 cached, 17 total
  Time:    35.022s


> @senecacdot/telescope@3.0.9 clean:root C:\OSD\telescope
> pnpm rimraf .turbo node_modules || pnpm rimraf node_modules

The system cannot find the path specified.
'rimraf' is not recognized as an internal or external command,
operable program or batch file.
 ELIFECYCLE  Command failed with exit code 1.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
ERROR: "clean:root" exited with 1.
The system cannot find the path specified.
 ELIFECYCLE  Command failed with exit code 1.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
```

RC-Lee commented:
Ok, I see my `node_modules` in root finally disappeared, probably had to wait a bit. 

I'll try again.

cindyledev commented:
> Ok, I see my `node_modules` in root finally disappeared, probably had to wait a bit.
> 
> I'll try again.

I'm not entirely sure why this happens, but when we run `pnpm clean:root` , it gets executed two times. On the first attempt, it does its job and deletes the `node_modules` directory as expected, but on the second attempt, it complains that it can't find `rimraf`, which makes sense because it was deleted in the first attempt. It's bizarre, so I just ignore that error message if I see the `node_modules` directory is deleted

RC-Lee reviewed (dismissed):
I have about a 3 minute wait time for when the command exits to when my `node_modules` in root disappears. But it works. 

As for vercel, I don't think its just this PR. It was suddenly not working last night, we tried to "fix" it in #3707. It builds in the PR, but then doesn't pass again when we merge the commit.
